packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx

```diff
@@ -32,11 +32,13 @@ export interface MeasureToolsUiProviderOptions {
   // If we check for sheet to 3d transformation when measuring in sheets
   enableSheetMeasurement?: boolean;
   stageUsageList?: string[];
+  additionalToolbarItems?: ToolItemDef[];
 }
 
 export class MeasureToolsUiItemsProvider implements UiItemsProvider {
   public readonly id = "MeasureToolsUiItemsProvider";
   private _props: RecursiveRequired<MeasureToolsUiProviderOptions>;
+  private _additionalToolbarItems: ToolItemDef[] = [];
```

Comment:
do we need this? we're gonna have 2 "places of truth"
_props would have additionalToolbarItems itself, but you set that to always be an empty array

Reply:
Moved additionalToolbarItems into _props only